test/runnable/inline.d

```diff
@@ -776,10 +776,30 @@ void test7625()
 {
     int x = foo7625(1);
     if (x != 1)
-	assert(0);
+        assert(0);
 }
 
 /**********************************/
+// 9785 partial fix
+
+void test9785()
+{
+        int j = 3;
+
+        void loop(scope const void function(int x) dg) {
```

Comment:
I never seen const delegates/functions in the real code. Even `void toString(scope void delegate(const(char))` is not written like that. Therefore this partial fix fixes nothing IMHO - upgrade to the whole thing?